packages/yoga/src/hooks/index.js

```diff
@@ -0,0 +1,3 @@
+import usePortal from './usePortal';
```

Comment:
If we want to make this `index` file easier to maintain, maybe we can change some things.

What about exporting the hook from the `usePortal.jsx` file as a named export, and just re-export it here, like this:

```js
// At hooks/usePortal.jsx

export const usePortal = () => { /* ... */ };

// At hooks/index.js

export * from './usePortal';
```

This way we only need to add 1 line to this file, and if we use something like [`glob`](https://www.npmjs.com/package/import-glob) in the future, it would be even easier. 😊 

What do you think?

Reply:
Great idea 👌